src/main/features/claim/helpers/taskListBuilder.ts

```diff
@@ -0,0 +1,42 @@
+import DraftClaim from 'drafts/models/draftClaim'
+import TaskList from 'drafts/tasks/taskList'
+import TaskListItem from 'drafts/tasks/taskListItem'
+import { Paths } from 'claim/paths'
+import { ResolveDispute } from 'drafts/tasks/resolveDispute'
+import { CompletingYourClaim } from 'drafts/tasks/completingYourClaim'
+import { ClaimAmount } from 'drafts/tasks/claimAmount'
+import { ClaimDetails } from 'drafts/tasks/claimDetails'
+import { YourDetails } from 'app/drafts/tasks/yourDetails'
+import { TheirDetails } from 'app/drafts/tasks/theirDetails'
+
+export class TaskListBuilder {
```

Comment:
That is good refactoring. Something I was thinking for some time however in my head I took it even further. I would like to have factory method which returns fully configured `TaskList` (not just list of `TaskListItem`). That would be used in the first place to render the task list (using common task list fragment shared across all parts of the app which iterates through model and renders both task list headlines as well as tasks itself). That would be used as well in guards and other places where task list has to be questioned. 

I understand that refactoring is beyond what user story defines so I am pointing this as guideline which you can follow if you wanna refactor it even more but cannot ask you to do within the story.